driver-examples/osgi/src/main/java/com/datastax/driver/osgi/impl/Activator.java

```diff
@@ -17,10 +17,13 @@
 
 import java.util.Hashtable;
 
+import com.sun.org.apache.xpath.internal.operations.Bool;
+import io.netty.channel.socket.SocketChannel;
```

Comment:
> Should we explicitly support shaded Netty + Netty customization?

No, we shouldn't.
The only thing I want to check is that having shaded classes in the public API doesn't break OSGI, as I explained in another comment.